tools/querytee/proxy.go

```diff
@@ -157,7 +157,7 @@ func NewProxy(cfg ProxyConfig, logger log.Logger, routes []Route, registerer pro
 	return p, nil
 }
 
-func (p *Proxy) Start() error {
+func (p *Proxy) Start(logger log.Logger) error {
```

Comment:
Why? We already pass the logger to `NewProxy()`. Passing it to the `Start()` looks an anti-pattern to me.

Reply:
My bad. I should have looked better the whole source, and not only the `Start` function. There is a field `logger` that we can use. Thank you